### DISCUSSION TITLE
Day difference in calucalations

### DISCUSSION THREAD

**Discussion by KrissyPerrine** [pwillworth/dfkreport#3] [Q&A] [answered]
Hi, Thank you so much for your report. In looking it over and trying to make sense of the jewel price when I purchased heroes this morning, I noticed that my hero 37096 was purchased at 16:32 UTC time and on your chart, it shows today's date 1/20/22. However, for the number to be correct at $13.30 cost of jewel, the date would have to be 16:32 UTC on 1/19/22. Now, for clarity, I have covid and mush brain, so this could be all wrong. But I'm showing on the chart for today that at 16:32 UTC the price of jewel was $14.41, not $13:30 which means I paid about $70 more than shown on the chart.

**Comment by pwillworth** [comment, marked as answer]
Hi! Please see caveats about prices on the [help page](https://dfkreport.cognifact.com/help.py) - particularly about prices only being pegged to daily data.  I have plans to enhance it later to have the prices pegged to a more granular level, but in the timeline of the bounty I was only able to get it to a daily approximation for prices.  If you are looking for it to be able to show the exact USD price of the thing when you bought, it's not capable of that at this time.  Sorry for the confusion!

**Reply by KrissyPerrine** [reply]
ok, no worries and thanks for all your hard work!!!!
